src/main/styles.css

```diff
@@ -1,55 +1,94 @@
-.nav_bar {
+.bp--navpage-container {
     display: grid;
-    grid-gap: 30px;
-    grid-template-columns: 200px repeat(6, 1fr);
+    grid-template-columns: repeat(2, [col] auto);
     background-color: #4A4848;
-    margin:12px;
-    text-transform: uppercase;
-    font-size: 10px;
-    align-items: center;
-}
-
-.login_box {
-    display: flex;
-}
-
-.nav_log {
-    flex: 1 0 auto;
-}
-
-.forgot_pass {
-    flex: 1 0 auto;
 }
 
-.new_account {
-    flex: 1 0 auto;
+.bp--company-logo {
+    display: grid;
+    grid-column: col;
+    text-transform: uppercase;
+    align-items: center;
+    padding-top: 30px;
+    padding-bottom: 30px;
+    text-decoration: none;
+    color: #CFCECE;
+    letter-spacing: 1px;
+    font-size: 150%;
 }
 
-.log_form_field {
+.bp--navigation-menubar {
+    grid-column: col 2;
     display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    background-color: #4A4848;
+    margin: 12px;
+    text-transform: uppercase;
+    font-size: 10px;
     align-items: center;
-    grid-template-columns: 1fr;
-    padding-top: 5px;
-    padding-bottom: 5px;
-    padding-left: 20px;
-    padding-right: 20px;
 }
 
-.nav_text {
+.bp--navigation-menuitem {
     //background-color: #7F26E9;
     padding-top: 30px;
     padding-bottom: 30px;
-    font-family: Arcon;
     text-decoration: none;
     color: #CFCECE;
     letter-spacing: 1px;
     font-size: 150%;
 }
 
-.nav_text:hover {
+.bp--navigation-menuitem:hover {
     color: #FFFFFF;
 }
 
-.main_header {
+.bp--nav-container {
     background-color: #4A4848;
+}
+
+.bp--hamburger-button {
+    font-size: 24px;
```

Comment:
Do we want to set a global font size somewhere as the "base"?

Reply:
Yeah, I definitely think that would be a good idea.